content/chapters/software-stack/lecture/slides/types-of-software.md

```diff
@@ -0,0 +1,62 @@
+---
+
+## Types of Software
+
+----
+
+### Usable vs Reusable
+
+* Software provides features by extending hardware
+* Features are used by:
+  * users (as interactive applications / processes)
+  * system (as non-interactive services / processes)
+  * other applications (as reusable libraries)
+* Software components
+  * Applications: usable
+  * Libraries: reusable
+
+----
+
+### Applications
+
+![Applications](media/applications.svg)
+
+----
+
+### Libraries
+
+![Libraries](media/libraries.svg)
+
```

Comment:
I feel like these images belong to the same picture. What if you merge them?

Reply:
I want to have a distinction of each of their role,